packages/backend-data/src/factory.test.ts

```diff
@@ -757,22 +692,8 @@ void describe('DataFactory', () => {
                 'Fn::Join': [
                   '',
                   [
-                    'arn:',
-                    {
-                      Ref: 'AWS::Partition',
-                    },
-                    ':appsync:',
-                    {
-                      Ref: 'AWS::Region',
-                    },
-                    ':',
-                    {
-                      Ref: 'AWS::AccountId',
-                    },
-                    // eslint-disable-next-line spellcheck/spell-checker
-                    ':apis/',
                     {
-                      'Fn::GetAtt': ['amplifyDataGraphQLAPI42A6FA33', 'ApiId'],
+                      'Fn::GetAtt': ['amplifyDataGraphQLAPI42A6FA33', 'Arn'],
```

Comment:
so `ApiId` is no longer available from the nested stack? Why is this not breaking change?

Reply:
It's not that it's not available.

Data construct just changed how do they obtain arn for policy statement.

I.e. instead of formatting arn themselves from other attributes they read this field now https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_appsync.CfnGraphQLApi.html#attrarn .

Reply:
which in turn affected the assertion. runtime behavior should be same.